### ISSUE TITLE
Component is not compatible with Vaadin 23.2

### ISSUE THREAD

**Issue by echarlus** [vaadin-component-factory/multi-combo-box-flow#23]
Hi,
first of all thanks for this great contribution ! I was happy to see that you released a fix for it work with Vaadin 23.1.X and I happily used it ;)
Now I'm trying to upgrade to 23.2 and the component breaks again. I had a quick look at the changes made to fix the issue on Vaadin 23.1.x but I could not find an obvious enhancement to make it work on 23.2 ...

**Comment by jcgueriaud1** [comment]
There is some internal changes in the combobox that makes this component incompatible in v23.2. One solution is to use the official multi combo box in v23.2. The API is very similar. You can also wait for an update of the component.

**Comment by echarlus** [comment]
Thanks for the reply. I'm currently looking at vcf-multiselect-combo-box/src/vcf-multiselect-combo-box.js and I see that it's indeed referring to deprecated vaadin-combo-box while it appears it should be using combo-box. I'm trying to change that to see if it automagically fixes the issue but I guess it won't be that easy. 
Unfortunately using the new Vaadin component does not work for me since I need the select all feature of this one ...

**Comment by jcgueriaud1** [comment]
Related to this issue https://github.com/vaadin-component-factory/multiselect-combo-box/issues/28
There is probably nothing to change on the Java side

**Comment by TatuLund** [comment]
Yes, probably the same fix is needed here than we did for another ComboBox variant https://github.com/vaadin-component-factory/vcf-auto-select-combobox/commit/ce7bea93d144dfaa12d15467f9033e47429bb17b
